src/experimental/build/commands/build/get_context.ts

```diff
@@ -3,17 +3,20 @@ import { join } from 'path';
 
 import { getContext as getStableContext } from '#commands/build/get_context';
 import { Context, WasmData } from '#experimental/utils/types';
+import { CanisterConfig } from '#experimental/utils/types';
 import { getCanisterId } from '#utils/dfx';
 import { AZLE_ROOT } from '#utils/global_paths';
-import { CanisterConfig } from '#utils/types';
 
 import { getConsumer } from './open_value_sharing/consumer';
 
 export async function getContext(
     canisterName: string,
     canisterConfig: CanisterConfig
 ): Promise<Context> {
-    const stableContext = await getStableContext(canisterName, canisterConfig);
+    const stableContext = await getStableContext(canisterName, {
```

Comment:
I think this is not correct, at runtime we need all of the values present that were in their dfx.json. Maybe cast things or something, or maybe getStableContext needs a type argument...probably not

Reply:
This comment was just a little misunderstanding, nothing to do here.